Fastfile.private

```diff
@@ -196,20 +208,22 @@ reflected in `Info.plist` during building."
       app_identifier: bundle_identifier,
       force: false,
       p12_password: p12_password,
-      output_path: output_path
+      output_path: output_path,
+      username: credentials[:account],
```

Comment:
I'd make the names match the expected ones: "username" -> "username", "team_id" -> "team_id".

Reply:
the thing is that sometime itc_team_id is team_id because the other team_id isn't necessary for that lane. Or sometimes is username and sometime I don't remember what else.
So I tried to give them an understandable name despite the context for what it will be used.
Like the itc_team is the team from itunes connect, and account si the account by which to login to apple.